examples/federated-connections/langchain/README.md

```diff
@@ -0,0 +1,89 @@
+# Step Up Auth for Tools with LangChain
+
+## Getting Started
+
+#### `.env` file
+
+```sh
+# Auth0
+AUTH0_DOMAIN="<auth0-domain>"
+AUTH0_CLIENT_ID="<auth0-client-id>"
+AUTH0_CLIENT_SECRET="<auth0-client-secret>"
+
+# API
+API_URL=http://localhost:8081/
+AUDIENCE=http://localhost:8081
+
+# OpenAI
+OPENAI_API_KEY="openai-api-key"
+
+# Langchain
+LANGGRAPH_API_URL="http://localhost:54367"
+```
+
+### How to run it
+
+1. Install dependencies. If you want to run with local dependencies follow root instructions.
+
+   ```sh
+   $ npm install
+   ```
+
+2. Running the API
+
+   ```sh
+   npm run start:api
+   ```
+
+3. Running the scheduler
+
+   ```sh
+   npm run scheduler:up
+   ```
+
+4. Running the example
+   ```sh
+   npm run dev
+   ```
+
+### How this works
+
+```mermaid
+sequenceDiagram
```

Comment:
This seems like a mistake from copy paste as this is a CIBA example and not FCAT